cluster/seq-cluster-docker-compose/compose.yaml

```diff
@@ -53,9 +96,10 @@ services:
     networks:
       - seqclusterdemo
     ports:
-      - 80:80
+      - "5777:80"
     environment: 
       - ACCEPT_EULA=Y
+      - SEQ_FEATURES_ENABLED=IngestOtlpMetrics
```

Comment:
Not a general-consumption production feature, but useful for low-volume cluster diagnostics.